packages/database/src/seed.ts

```diff
@@ -0,0 +1,240 @@
+// SPDX-FileCopyrightText: 2023 Double Open Oy
+//
+// SPDX-License-Identifier: MIT
+
+import crypto from "crypto";
+import fs from "fs";
+import path from "path";
+import readline from "readline";
+import { PrismaClient } from "@prisma/client";
+import { objectExistsCheck, uploadFile } from "s3-helpers";
+
+const prisma = new PrismaClient();
+
+async function main() {
+    const testUserSalt = crypto.randomBytes(16);
+    const testUserHashedPassword = crypto.pbkdf2Sync(
+        "test",
+        testUserSalt,
+        310000,
+        32,
+        "sha256",
+    );
+    const testUser = await prisma.user.upsert({
+        where: {
+            username: "test",
+        },
+        update: {},
+        create: {
+            username: "test",
+            token: "test_token",
+            salt: testUserSalt,
+            hashedPassword: testUserHashedPassword,
+        },
+    });
+    console.log(testUser);
+
+    console.log("Uploading test files to object storage");
+
+    const files = fs.readdirSync(path.join(__dirname, "./test_data/files"));
```

Comment:
This could be a little cleaner if the seed files for S3 were stored in an archive and extracted to temp directory for the seed.